src/pages/ManageComputeHost/ActivateComputeHost.js

```diff
@@ -0,0 +1,130 @@
+      overallStatus: STATUS.NOT_STARTED
+    };
+  }
+
+  getSteps() {
+    return [
+      {
+        label: translate('server.activate.text', this.props.operationProps.target.id),
+        playbooks: [`${ARDANA_START_PLAYBOOK}.yml`],
+        payload: {
+          limit: this.props.operationProps.target.hostname
+        }
+      },
+      {
+        label: translate('server.activate_service.text', this.props.operationProps.target.id),
+        playbooks: [NOVA_ACTIVATE]
+      }
+    ];
+  }
+
+  getPlaybooks() {
+    return [
+      {
+        name: ARDANA_START_PLAYBOOK,
+        payload: {
+          limit: this.props.operationProps.target.hostname
+        }
+      },
+      {
+        name: NOVA_ACTIVATE,
+        action: ::this.activateNova
+      }
+    ];
+  }
+
+  async activateNova(logger) {
+    const { hostname } = this.props.operationProps.target,
+      apiUrl = `/api/v2/compute/services/${hostname}/enable`;
+    logger(`PUT ${apiUrl}`);
+    try {
+      let responce = await putJson(apiUrl),
+        logMsg = `Got response from enabling compute services for compute host ${hostname}`;
+      logProgressResponse(logger, responce, logMsg);
+    } catch(error) {
+      if(error.status === 410) {
+        logger(`Warning: No compute service found for compute host ${hostname}, continue...`);
+      }
+      const logMsg = `Got error responce from enabling compute services for compute host ${hostname}`;
+      this.updatePageStatus(STATUS.FAILED);
+      this.setState({
+        processErrorBanner: translate('server.activate.error', hostname, error.toString())
+      });
+      logProgressError(logger, error, logMsg);
+    }
+  }
+
+  updatePageStatus(status) {
+    let state = {
+      overallStatus: status,
+    };
+    if(status === STATUS.FAILED && ! this.state.processErrorBanner) {
+      state.processErrorBanner =
+        translate('server.activate.error', this.props.operationProps.target.hostname, 'UNKNOWN');
+    }
+    this.setState(state);
+  }
+
+  render() {
+    const failed = this.state.overallStatus === STATUS.FAILED,
+      steps = this.getSteps(),
+      playbooks = this.getPlaybooks();
+    return (
+      <div className='wizard-page'>
+        <LoadingMask show={this.props.wizardLoading || this.state.loading}/>
+        <div className='content-header'>
+          {this.renderHeading(translate())}
+        </div>
+        <div className='wizard-content'>
+          <PlaybookProgress
+            updatePageStatus={::this.updatePageStatus} updateGlobalState={this.props.updateGlobalState}
+            playbookStatus={this.props.playbookStatus} steps={steps} playbooks={playbooks}/>
+          <If condition={failed}>
+            <div className='banner-container'>
+              <ErrorBanner message={this.state.processErrorBanner}
+                show={true}/>
+            </div>
+          </If>
+        </div>
+        {this.renderNavButtons(failed)}
+      </div>
+    );
+  }
+}
+
+export default ActivateComputeHost;
```

Comment:
I think when deactivate...the progress page you use ansible host name, but in activate progress...you use the server ID...think it might be better use ansible host name + server ID ...otherwise it could confuse users.